.eslintrc.js

```diff
@@ -51,21 +51,24 @@ module.exports = {
       ],
     ],
     'import/no-unresolved': ERROR,
-    'import/no-extraneous-dependencies': [ERROR, {"includeTypes": true}],
-    "react/require-default-props": [WARNING, {
-      "functions": "defaultArguments"
-    }],
-    'react/jsx-filename-extension': [WARNING, { extensions: ['.js', '.jsx', '.ts', '.tsx'] }]
+    'import/no-extraneous-dependencies': [ERROR, {includeTypes: true}],
+    'react/require-default-props': [
+      WARNING,
+      {
+        functions: 'defaultArguments',
+      },
+    ],
+    'react/jsx-filename-extension': [
+      WARNING,
+      {extensions: ['.js', '.jsx', '.ts', '.tsx']},
+    ],
   },
   settings: {
     'import/parsers': {
       '@typescript-eslint/parser': ['.ts', '.tsx'],
     },
     'import/resolver': {
-      'webpack': true,
```

Comment:
why changes here?

Reply:
Though Docusaurus uses webpack, it is actually used internally with no exposed webpack config. It exposes things through javascript modules.